gstreamer-pbutils/build.rs

```diff
@@ -0,0 +1,34 @@
+fn main() {
+    manage_docs();
+}
+
+#[cfg(any(feature = "embed-lgpl-docs", feature = "purge-lgpl-docs"))]
+fn manage_docs() {
+    extern crate stripper_lib;
+    use std::io;
+
+    let path = "src";
+    let ignores: &[&str] = &[];
+
+    stripper_lib::loop_over_files(
+        path.as_ref(),
+        &mut |w, s| stripper_lib::strip_comments(w, s, &mut io::sink(), true),
+        &ignores,
+        false,
+    );
+
+    #[cfg(feature = "embed-lgpl-docs")]
+    {
+        let docs = include_str!("../docs/gstreamer-pbutils/docs.md");
```

Comment:
This is why CI fails, you need to create this file with gir running in "-m doc" mode